### PR TITLE
feat(fluent): add module parameter to with connector

### PR DESCRIPTION
**Pull request by schotime**

### CODE REVIEW

File: packages/demos/todomvc-fluent/tslint.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (henri-hulski) [inline packages/demos/todomvc-fluent/tslint.json]: I think this file should be removed from repo.

**Comment by schotime** [comment]
@henri-hulski try running the fluent demo without it. You cannot. It needs to be there.

**Comment by henri-hulski** [comment]
@schotime  That's interesting. Does it use tslint somewhere?
Will take a look when I'm back home.

**Comment by henri-hulski** [comment]
Yep seems like tslint is somehow part of CRA.
Didn't know that so I removed that file.
@schotime  I think we should revert 70f2f4d2ce6f7528057 in this case.

**Comment by henri-hulski** [comment]
Maybe in a separate commit.

**Comment by schotime** [comment]
Reverted and updated PR

**Comment by henri-hulski** [comment]
Great!

**Pull request merged**